test/gtest/ucs/test_config.cc

```diff
@@ -684,3 +689,21 @@ UCS_TEST_F(test_config, test_config_file) {
     /* Overriding INI file by env vars */
     EXPECT_EQ(std::string("Ford"), std::string(opts->brand));
 }
+
+UCS_TEST_F(test_config, test_config_file_parse_files) {
+    /* coverity[tainted_string_argument] */
+    ucs::scoped_setenv ucx_config_dir("UCX_CONFIG_DIR", TEST_CONFIG_DIR);
+    
+    car_opts_t opts;
+    ucs_status_t status;
+
+    ucs_config_parse_config_file(TEST_CONFIG_DIR, TEST_CONFIG_FILE, 1);
+    ucs_config_parse_config_files();
```

Comment:
isn't this a duplication? why need ucs_config_parse_config_files()?

Reply:
That's the point of the test. Line 700 parses config, line 701 overrides it. Then I check it's indeed overridden.

Reply:
shall we check return status from ucs_config_parse_config_file(TEST_CONFIG_DIR,..) ? and that after the first call the price is 200?

Reply:
Done